packages/EasyCore/tests/Bridge/Symfony/ApiPlatform/Normalizers/CarbonImmutableNormalizerTest.php

```diff
@@ -8,15 +8,15 @@
 use Carbon\CarbonImmutable;
 use Carbon\CarbonInterface;
 use Carbon\CarbonTimeZone;
-use EonX\EasyCore\Bridge\Symfony\ApiPlatform\Normalizers\CarbonNormalizer;
+use EonX\EasyCore\Bridge\Symfony\ApiPlatform\Normalizers\CarbonImmutableNormalizer;
 use EonX\EasyCore\Tests\Bridge\Symfony\AbstractSymfonyTestCase;
 use stdClass;
 use Symfony\Component\Serializer\Exception\InvalidArgumentException;
 
 /**
- * @covers \EonX\EasyCore\Bridge\Symfony\ApiPlatform\Normalizers\CarbonNormalizer
+ * @covers \EonX\EasyCore\Bridge\Symfony\ApiPlatform\Normalizers\CarbonImmutableNormalizer
  */
-final class CarbonNormalizerTest extends AbstractSymfonyTestCase
+final class CarbonImmutableNormalizerTest extends AbstractSymfonyTestCase
```

Comment:
Pls rewrite test to test denormalization instead of normalization.

Reply:
Done